datafusion/proto/src/logical_plan/from_proto.rs

```diff
@@ -1588,7 +1588,7 @@ pub fn parse_expr(
                     Ok(character_length(parse_expr(&args[0], registry)?))
                 }
                 ScalarFunction::Chr => Ok(chr(parse_expr(&args[0], registry)?)),
-                ScalarFunction::InitCap => Ok(ascii(parse_expr(&args[0], registry)?)),
+                ScalarFunction::InitCap => Ok(initcap(parse_expr(&args[0], registry)?)),
```

Comment:
💯  for testing @Weijun-H  -- your test also found two other functions with similar problems to https://github.com/apache/arrow-datafusion/pull/8847 👏 

I took the liberty of fixing them